bin/reth/src/commands/p2p/mod.rs

```diff
@@ -223,3 +209,17 @@ impl Command {
         Ok(())
     }
 }
+
+fn ipv4(ip: IpAddr) -> Option<Ipv4Addr> {
+    match ip {
+        IpAddr::V4(ip) => Some(ip),
+        IpAddr::V6(_) => None,
+    }
+}
+
+fn ipv6(ip: IpAddr) -> Option<Ipv6Addr> {
```

Comment:
looks like this is only used once?

can we inline this simple match and remove these functions?